.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Thanks for contributing to Altair!
```

Comment:
I like the text! Could we wrap it in `<!-- Here the text -->` so that we don't even have to delete the message before submitting the PR? See how Ibis handles this: https://raw.githubusercontent.com/ibis-project/ibis/main/.github/PULL_REQUEST_TEMPLATE.md

I think I've also seen this in other repos as well.

Reply:
Yeah, I have seen this in other repos too, however I'm a bit split on whether it is preferable. The reason I kept it as uncommented text here is so that it can be read in the rich preview; I find that the links makes the comment cluttered and hard to parse.

For the issues, the answer is to use "Issue forms" instead of the regular templates. This renders links and formatting nicely while not including it in the actual issue. In my latest commit I switched both our issue templates over, [you can see a demo in this other repo](https://github.com/joelostblom/test-pr-templates/issues/new/choose) I made.

![image](https://github.com/altair-viz/altair/assets/4560057/0bcaa9bd-22eb-4118-86bb-ae27144816e0)

![image](https://github.com/altair-viz/altair/assets/4560057/5d00bb1f-672d-4bb2-b217-7beec029c118)

Unfortunately, [forms don't exist for PR ](https://github.com/orgs/community/discussions/4881).